File: app/src/main/java/moe/tlaster/openween/core/model/BaseListModel.java
package moe.tlaster.openween.core.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Tlaster on 2016/8/26.
 */
public class BaseListModel {
    @SerializedName("previous_cursor")
    private String mPreviousCursor;
    @SerializedName("next_cursor")
    private String mNextCursor;
    @SerializedName("total_number")
    private int mTotalNumber;

    public String getPreviousCursor() {
        return mPreviousCursor;
    }

    public void setPreviousCursor(String previousCursor) {
        mPreviousCursor = previousCursor;
    }

    public String getNextCursor() {
        return mNextCursor;
    }

    public void setNextCursor(String nextCursor) {
        mNextCursor = nextCursor;
    }

    public int getTotalNumber() {
        return mTotalNumber;
    }

    public void setTotalNumber(int totalNumber) {
        mTotalNumber = totalNumber;
    }
}

File: app/src/main/kotlin/moe/tlaster/openween/core/model/BaseListModel.kt
package moe.tlaster.openween.core.model

import com.google.gson.annotations.SerializedName

/**
 * Created by Tlaster on 2016/8/26.
 */
open class BaseListModel {
    @SerializedName("previous_cursor")
    var previousCursor: String? = null
    @SerializedName("next_cursor")
    var nextCursor: String? = null
    @SerializedName("total_number")
    open var totalNumber: Int = 0
}
